### PR TITLE
Crud server tests

### PR DESCRIPTION
**Pull request by kavithaii**
Compare and review with master branch before merging into master

### CODE REVIEW

File: crud-server/src/model.test.js [hunk 1]
```diff
@@ -607,27 +607,6 @@ describe('PraaS', () => {
...
-                expect(e.errors[0].path).to.equal('suriObjectKey');
-                done();
-              });
-          })
-          .catch(e => done(e));
-      });
-
       it('should not allow empty suriObjectKey', done => {
         helpers.makeCuri('td')
           .then(curi => helpers.fakeConduit({ userId: user.id, curi }))
```
Comment (m5nv) [inline crud-server/src/model.test.js]: This test case is also redundant when suriObjectKey can be null - meaning it's optional.

Comment (m5nv) [reply crud-server/src/model.test.js]: The entire section consisting of tests for suriObjectKey validity can be removed.

File: crud-server/src/routes.test.js [hunk 2]
```diff
@@ -126,89 +126,703 @@ describe('Praas REST API', () => {
...
+            const ct = await helpers.fakeConduit();
+            ct.suriType = 'random';
+            const res = await Api()
+              .post(`/conduits`)
+              .set('Authorization', `Token ${jakeUser.token}`)
+              .send({ conduit: ct });
+            expect(res.status).to.equal(422);
+            expect(res.body.error.errors[0].path).to.equal('suriType');
+          });
+        });
```
Comment (m5nv) [inline crud-server/src/routes.test.js:315]: Please change the test description to 'should reject unsupported service types'.
Also, please remove backslash to escape single quote. Replace either with double quotes or use back tick.

Comment (kavithaii) [reply crud-server/src/routes.test.js:315]: I have to change the test as `it('should reject unsupported service types "Airtable", "Google sheets", or "Smart sheet"', async() => {`

**Comment by m5nv** [comment]
Too many changes to do via github. Let's bring it into master and do a pair programming session to fix the remaining issues.
Marking it as LGTM, please squash merge and let's continue in Code.

**Pull request merged**